spring-cloud-dataflow-composed-task-runner/src/main/java/org/springframework/cloud/dataflow/composedtaskrunner/properties/ComposedTaskProperties.java

```diff
@@ -164,7 +164,7 @@
 	 * ComposedTaskRunner instance can only be executed once with a given set
 	 * of parameters, if true it can be re-executed.
 	 */
-	private boolean incrementInstanceEnabled = false;
+	private boolean incrementInstanceEnabled = true;
```

Comment:
Probaboy need a SCDF variable that enables or disables whether this is true or not.  This could affect folks who already have an expectation that incrementInstanceEnabled is false.

Reply:
Since SCDF owns the CTR and having this flag still be overridable, I think it can be ok for the existing users who have already set this flag to `true` or `false`.